index.html

```diff
@@ -0,0 +1,526 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="./assets/bootstrap/bootstrap.css" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Merriweather:300|Poppins:300,600" rel="stylesheet">
+    <link href="./public/stylesheets/style.css" rel="stylesheet">
+  </head>
+
+  <body>
+  <div id="main">
+
+    <!--1) GREEN BACKGROUND START-->
+    <div class="bg-green">
+
+    <!--HEADER-->
+      <div class="content-wrap lrg-rl-padding" id="header">
+        <header>
+          <p class="text -x-small -color-dark -caps -wide-spacing -bold centered">A Viget Exploration</p>
+          <img class="header-img" src="./public/img/title.png" alt="Improving Medical Adherence">
+        </header>
+        <div class="pills -yellow"></div>
+        <div class="pills -multiple"></div>
+        <div class="pills -double"></div>
+        <div class="pills -purple"></div>
+      </div>
+
+
+    <!--INTRO SECTION-->
+      <div class="content-wrap lrg-rl-padding bg-white" id="intro">
+        <section>
+          <p class="text -medium -serif">Patients who don’t take medication as prescribed are at risk of worsening health, increased hospitalization, and a greater chance of mortality.</p>
+          <p class="text -small">Just in the U.S., non-adherence has been attributed to over $100 billion in avoidable healthcare costs.<cite><a href="">1</a></cite> In developed countries, among patients with chronic illness, approximately 50% do not take medications as prescribed.<cite><a href="">2</a></cite></p>
+
+          <div class="quote-container">
+            <div class="pills bottle-1"></div>
+            <div class="pills bottle-2"></div>
+            <div class="pills bottle-3"></div>
+            <div class="quote">
+              <p class="text -small -serif -color-accent">“Increasing the effectiveness of adherence interventions might have a greater impact on the health of the population than any improvement in specific medical treatment.”</p>
+              <footer class="text -x-small -caps -bold">World Health Organization, 2003<cite><a href="">3</a></cite></footer>
+            </div>
+          </div>
+        </section>
+    </div>
+
+    <!--DIAGRAM SECTION-->
+    <div class="content-wrap">
+      <section class="diagram">
+        <div class="centered">
+          <p class="text -x-small -color-dark -caps -wide-spacing -bold">Why People Don't Adhere</p>
+          <p class="text -large -serif -color-dark">Myriad factors conspire to prevent people from sticking to their treatment programs. But a few are most telling.</p>
+        </div>
+        <div class="diagram-details">
+          <div class="diagram-detail centered">
+            <!--transparent borders-->
+            <div class="icon -borders"></div>
+            <!--background-color-->
+            <div class="icon -color">
+              <!--img-->
+              <img class="resize" src="./public/img/icon-algorithmic.png">
```

Comment:
Accessibility tip: on images that don't have or need `alt` text, use `alt=""`. This tells a screen reader "nothing to read here" and it skips by. Otherwise a screen reader will read whatever it can. In this case, it would be the file name.